tests/cases/fourslash/extract-method-in-anonymous-function-declaration.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path='fourslash.ts' />
 
 ////export default function() {
-////    /*start*/0/*end*/
+////    /*start*/0/*end*/;
 ////}

```

Comment:
I see, these are all tests which use the writer's heuristic for semis - and they all now get the semi because it doesn't have proof that the file isn't using them. Makes sense.